README.md

```diff
@@ -161,3 +164,11 @@ Client contracts which initiate a request and receive a fulfillment can be modif
 ## Simulating Requests
 
 An end-to-end request initiation and fulfillment can be simulated using the `npx hardhat functions-simulate` command. This command will report the total estimated cost of a request in LINK using the latest on-chain gas prices. Costs are based on the amount of gas used to validate the response and call the client contract's `fulfillRequest` function, plus a flat fee. Please note that actual request costs can vary based on gas prices when a request is initiated on-chain.
+
+## Off-chain Secrets
+
+Instead of using encrypted secrets stored directly on the blockchain, encrypted secrets can also be hosted on URLs and be fetched by the DON nodes when a request is initiated.  To generate the encrypted secrets JSON file, run the command `npx hardhat functions-build-offchain-secrets --network network_name_here`.  This will output the file `offchain-secrets.json` which can be hosted at any URL such as Github or S3.
+Then, set `secretsLocation` to `Location.Remote` in `Functions-request-config.js` and enter the URL(s) where the secrets are hosted into `secretsURLs`.  Multiple URLs can be entered as a fallback in case any of the URLs are offline.  Each URL should host the exact same JSON file.  The tooling will automatically pack the secrets URL(s) into a space-separated string and encrypt the string using the DON public key so no 3rd party can view the URLs.  Finally, this encrypted string of URLs is used in the `secrets` parameter when making an on-chain request. 
```

Comment:
Starting with "Then" feels like I missed some first step already. Maybe start with "In order to use off-chain secrets, ..."